pub-matic-htb.js

```diff
@@ -148,20 +289,26 @@ function PubMaticHtb(configs) {
          */
 
         /* ---------------------- PUT CODE HERE ------------------------------------ */
-        var queryObj = {};
-        var callbackId = System.generateUniqueId();
-
-        /* Change this to your bidder endpoint.*/
-        var baseUrl = Browser.getProtocol() + '//someAdapterEndpoint.com/bid';
-
-        /* ---------------- Craft bid request using the above returnParcels --------- */
-
-
+        var payload = {},
+        callbackId = System.generateUniqueId(),
+        //baseUrl = Browser.getProtocol() + '//hbopenbid.pubmatic.com/translator?';
+        baseUrl = '//hbopenbid.pubmatic.com/translator?';
+        payload = { 
+            id: '' + new Date().getTime(), // str | mandatory
+            at: Constants.AUCTION_TYPE, // int | mandatory
+            cur: [Constants.CURRENCY], // [str] | opt
+            imp: __populateImprObject(returnParcels), // obj | mandatory - pending
+            site: __populateSiteObject(returnParcels[0].pubId), //// obj | opt
```

Comment:
Since the adapter is set to support SRA, returnParcels can have length > 1. Is using returnParcels[0] expected?

Reply:
since each return parcel will have the same value for pubdId, device information and other common data such as lat, lon, yob, we are passing only the 1st return parcel.